Resources/Prototypes/Entities/Mobs/NPCs/flesh.yml

```diff
@@ -0,0 +1,162 @@
+﻿- type: entity
+  parent: SimpleMobBase
+  id: BaseMobFlesh
+  name: flesh construct
+  description: Looks messy.
```

Comment:
would be cooler if this was more descriptive (AKA ask chatgpt for "horrible meat monster description" as inspiration :heck:)